lib/excon/ssl_socket.rb

```diff
@@ -101,7 +100,7 @@ def initialize(data = {})
         @socket.write(request)
 
         # eat the proxy's connection response
-        Excon::Response.parse(self, { :expects => 200, :method => "CONNECT" })
+        Excon::Response.parse(self,  'expects' => 200, 'method' => 'CONNECT')
```

Comment:
Same here, symbols seem appropriate since they will appear frequently so we can not have to reallocate.